docs/policy/container-release.md

```diff
@@ -17,32 +17,43 @@ Similar to our existing RPM infrastructure, container image sources, build logs,
+OSG Software container images will be built at least weekly and tagged with one of the following:
 
-| Tag       | Description                                                                                                                                                       |
-|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `fresh`   | Images that build successfully and pass automated tests. Intended for users that need the latest fixes and features.                                              |
-| timestamp | Each `fresh` image  is also tagged with a timestamp reflecting their build date and time. Intended for rollback in case of issues with the current `fresh` image. |
-| `stable`  | `fresh` images that pass acceptance testing; approved by the Release Manager. Intended for stable production use.                                                 |
+| Tag                            | Description                                                                                                              |
+|--------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| `testing`                      | For the latest fixes and features. Built with RPMs from the `osg-testing` and/or `osg-upcoming-testing` Yum repositories |
+| `release`                      | For production use. Built with RPMs from the `osg-release` and/or `osg-upcoming-release` Yum repositories                |
+| `rc.testing.*`, `rc.release.*` | Release candidate images; see the [Validation section below](#validation) for details                                    |
 
-### Cleanup  ###
+Each newly built image from the table above will also be tagged with a `-<TIMESTAMP>` suffix to allow for rollback in
+case of broken images.
+
+### Cleanup ###
 
 The OSG will keep images based on the [Docker Hub image retention policy](https://www.docker.com/pricing/retentionfaq).
 In summary, container images that have not been pulled or updated in 6 months will be removed.
 
+Validation
+----------
 
-Announcements
--------------
+OSG Software container images consist of RPMs for OSG services that are tested through
+[existing release processes](software-release.md) as well as scripts and configuration specific to the container
+implementation of the service.
+New container images limited to RPM updates undergo additional automated testing before being published.
 
-Container images that have been tagged as **stable** will be noted in the OSG release notes and announcements.
+In order to test changes to container-specific scripts or configuration, OSG Software performs automated tests and
+coordinates testing of release candidate images (e.g. `rc.testing.*`, `rc.testing.*`) before applying these changes to
```

Comment:
What kinds of things will we have for the `*`?

Reply:
Dunno, branch name? I figured this could be somewhere we use [environment secrets](https://github.blog/changelog/2020-12-15-github-actions-environments-environment-protection-rules-and-environment-secrets-beta/):

1. Arbitrary user submits PR, it runs any validation tests but doesn't push `rc.testing.branchname` or `rc.release.branchname` tags to Docker Hub
2. Upon review from someone in OSG Technology, we approve usage of the environment secrets and push the RC tags to Docker Hub
3. Upon verification of the RC tag from another user, we merge the PR into the main branch

It gets a little sticky if there's a large time differential between 2 and 3 because the resultant images from 3 have a higher chance of having different RPMs, causing potential breakages.